### PR TITLE
Port precision_conj_bscore

### PR DESCRIPTION
**Pull request by Bitseat**

### CODE REVIEW

File: opencog/moses/scoring/precision_bscore.cc [hunk 1]
```diff
@@ -450,388 +453,438 @@ behavioral_score precision_bscore::exact_selection(const scored_combo_tree_set&
...
+	score_t sao = 0.0;     // sum of all active outputs (in the boolean case)
+
+	atomese::Interpreter interpreter(moses::compressed_value_key);
+	const ValuePtr result = interpreter(handle);
+	auto link_result = LinkValueCast(result)->value();
+	int i = 0;
+	for (const CTable::value_type &vct : ctable) {
+		// vct.first = input vector
+		// vct.second = counter of outputs
+		if (HandleCast(link_result.at(i))->get_type() == TRUE_LINK) {
```
Comment (ngeiswei) [inline opencog/moses/scoring/precision_bscore.cc]: Better use `bool_value_to_bool` defined in

https://github.com/singnet/asmoses/blob/998557895ad4e39ea43a80fbef2f5eb75351640c/opencog/utils/valueUtils.h#L12-L15

**Comment by ngeiswei** [comment]
@Bitseat you also need to rebase onto the master to remove the conflict. Looks good otherwise.

**Comment by Bitseat** [comment]
Done.

**Comment by ngeiswei** [comment]
@Bitseat you still have some conflict to resolve.

File: opencog/moses/scoring/precision_bscore.h [hunk 2]
```diff
@@ -217,7 +217,8 @@ struct precision_conj_bscore : public bscore_base
 	precision_conj_bscore(const CTable &_ctable, float hardness,
 	                      bool positive = true);
 
-	behavioral_score operator()(const combo_tree &tr) const;
+    behavioral_score operator()(const combo_tree& tr) const;
+    behavioral_score operator()(const Handle& handle) const;
```
Comment (ngeiswei) [inline opencog/moses/scoring/precision_bscore.h]: Better be smart tabs.

**Comment by ngeiswei** [comment]
@Bitseat it looks like `tests/moses/scoring/CMakeLists.txt` is missing the configuration to run `precision_conj_bscoreUTest`.

**Comment by Bitseat** [comment]
I hope it's ready now. sorry.

**Comment by ngeiswei** [comment]
Looks good, thanks.

**Pull request merged**